src/components/fileUploader/index.tsx

```
@@ -51,15 +51,17 @@ const FileUploader: React.FC<FileUploaderProps> = ({
   });
 
   const { isOpen, onOpen, onOpenChange } = useDisclosure();
-  const { acceptedFiles, fileRejections, getInputProps, open } = useDropzone({
-    accept: acceptedFileTypes.reduce((acc, fileType) => {
+  const { acceptedFiles, getInputProps, open } = useDropzone({
+    accept: props.acceptedFileTypes.reduce((acc, fileType) => {
       return { ...acc, [fileType]: [] };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)